dask/dataframe/io/sql.py

```diff
@@ -0,0 +1,60 @@
+import pandas as pd
+import numpy as np
+from dask import delayed
+from dask.dataframe import from_delayed
+
+
+def read_sql_table(table, uri, partitions, index_col, limits=None, columns=None,
+                   **kwargs):
+    """
+    Create dataframe from an SQL table.
+
+    Parameters
+    ----------
+    table : string
+        Table name
+    uri : string
+        Full sqlalchemy URI for the database connection
+    partitions : int or list of values
+        Number of partitions, or list of partition edges (number of partitions will be one less than length of list)
+    index_col : string
+        Column which becomes the index, and defines the partitioning. Should
+        be a indexed column in the SQL server, and numerical.
+        Could be a function to return a value, e.g., ROW_NUMBER().
+    limits: 2-tuple or None
+        Manually give upper and lower range of values; if None, first fetched max/min from the DB. Upper limit is
+        non-inclusive.
```

Comment:
we may be able to reuse this longer term as 

[start0, start1, start2, ... end] ?